src/features/vars-schema/env-vars.ts

```diff
@@ -73,6 +77,14 @@ export const SUPERVISOR_CONFIG_VAR_PROPERTIES: {
 		description: 'Enable / Disable logs from being sent to balena API',
 		default: 'true',
 	},
+	RESIN_SUPERVISOR_REPORT_INTERVAL: {
+		type: 'integer',
+		description:
+			'Define how frequently the device reports its current state to the cloud in milliseconds',
+		default: DEFAULT_SUPERVISOR_REPORT_INTERVAL,
+		minimum: DEFAULT_SUPERVISOR_REPORT_INTERVAL,
+		maximum: 86400000,
```

Comment:
this line could be changed to prevent large report intervals...maybe 24 hours ?

Reply:
```suggestion
		maximum: 24 * HOURS, // import HOURS from src/lib/config.ts
```